src/modules/LandingPage/Invitation.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  Text,
+  useBreakpointValue,
+} from '@chakra-ui/react';
+import getConfig from 'next/config';
+
+const { publicRuntimeConfig } = getConfig();
```

Comment:
CMIIW, I think we can't called this on modules/component level, I don't quite remember whats the reason but on time I do this and someone said that I should called this on `page` level and pass it on props. Although I don't remember the main reason, I think its better on `page` because we don't need to call this function again later if other modules/components on the same page need it.

Reply:
I agree, for now, I'll change for this seeking a short-term solution. 

Next, I'm going to make a new PR to decompose all components and make it more atomic for a long-term solution